README.md

```diff
@@ -26,15 +26,17 @@ The app combines a variety of different Google technologies, such as:
 ## Setup and installation
 
 1. Clone the repository.
-2. Create a [Firebase project](https://firebase.google.com/products/firebase-ai-logic) and
+1. Create a [Firebase project](https://firebase.google.com/products/firebase-ai-logic) and
    generate a `google-services.json` file.
    Place the file in the app folder: `app/google-services.json`. Be sure to enable [Vertex AI API](https://console.cloud.google.com/apis/library/aiplatform.googleapis.com).
    Ensure to also enable [AppCheck](https://console.firebase.google.com/project/_/appcheck) on your Firebase project to prevent API abuse.
 
-3. This project makes use of remote config on Firebase too, you can import the [Firebase Remote config](https://firebase.google.com/docs/remote-config) settings from 
+1. This project makes use of remote config on Firebase too, you can import the [Firebase Remote config](https://firebase.google.com/docs/remote-config) settings from 
 [`remote_config_defaults.xml`](core/network/src/main/res/xml/remote_config_defaults.xml)
 
-4. If you'd like to change the font that the app renders with, an optional spec can be placed in
+1. In [`remote_config_defaults.xml`](core/network/src/main/res/xml/remote_config_defaults.xml), update the value of [`use_imagen`](core/network/src/main/res/xml/remote_config_defaults.xml#L40) to `true`. 
```

Comment:
I'd move this step before import on the console.

Reply:
i consolidated it with the import step